web/src/components/typography/helper.ts

```diff
@@ -14,6 +14,7 @@ export type TextColorType =
   | 'inverse'
   | 'placeholder'
   | 'active'
+  | string
```

Comment:
sure that this is a good idea? this will not always work?

Reply:
Thanks! Added this because titleProps breaks webpack build because of wrong typing for a color property, as titleProps expecting types as for `HTMLElement` from React. Fixed this, by adding `color="primary"` to properties of <Headline>